securedrop/journalist_templates/col.html

```diff
@@ -93,18 +91,18 @@
 
     </form>
   {% else %}
-    <p><br>{{ gettext('No documents to display.') }}</p>
+    <p><br>{{ gettext('No submissions to display.') }}</p>
```

Comment:
if we are using the term `messages and/or files` above we should be consistent one way or another with that language. I would suggest changing L79 to `submissions` as it's a more straightforward change from `documents`.

Reply:
agreed. changed.